compose.go

```diff
@@ -76,6 +76,56 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	composeCmd.Flags().BoolVar(&conflictsVerbosity, "conflicts-verbosity", false, "Log files conflicts")
 	composeCmd.Flags().BoolVar(&clean, "clean", false, "Remove .compose dir on start")
 	composeCmd.Flags().BoolVar(&interactive, "interactive", true, "Interactive mode allows to submit user credentials during action")
+
+	composeDependency := &compose.Dependency{}
+	var createNew bool
+	var addCmd = &cobra.Command{
+		Use:   "compose:add",
+		Short: "Add new entry to plasma-compose",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.AddPackage(createNew, composeDependency, p.wd)
+		},
+	}
+
+	var updateCmd = &cobra.Command{
+		Use:   "compose:update",
+		Short: "Update plasma-compose entry",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.UpdatePackages(p.wd)
+		},
+	}
+
+	var toDeletePackages []string
+	var deleteCmd = &cobra.Command{
+		Use:   "compose:delete",
+		Short: "Remove entry from plasma-compose",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.DeletePackages(toDeletePackages, p.wd)
+		},
+	}
+
+	addCmd.Flags().BoolVarP(&createNew, "allow-create", "", false, "Create plasma-compose if not exist")
+	addCmd.Flags().StringVarP(&composeDependency.Name, "name", "", "", "Name of the package")
```

Comment:
Would be great to throw an error if url or package name is already present in plasma-compose: